tests/__init__.py

```diff
@@ -0,0 +1,24 @@
+def elasticsearch_running():
```

Comment:
I don't think any of the tests should actually require elastic search to be running. We normally patch methods in the libraries like `connect` or something similar. When elastic search is running, the tests add indexes and modify the data which may undesirable.